onboarding/tests/test_fibonacci.py

```diff
@@ -1,5 +1,7 @@
 import pytest
-from ..fibonacci import fibonacci_term
```

Comment:
Keep this. It will work. No need to use sys. To get to the fibonacci file

Reply:
should be fixed now